scripts/generate-redstone-data.ts

```
@@ -0,0 +1,231 @@
+  adapterContractType: string;
+  dataServiceId: string;
+  priceFeeds: {
+    [key: string]: {
+      priceFeedAddress: string;
+      updateTriggersOverrides?: {
+        deviationPercentage?: number;
+        timeSinceLastUpdateInMilliseconds?: number;
+        [key: string]: any;
+      };
+    };
+  };
```

ESLint

[error] A record is preferred over an index signature.

(@typescript-eslint/consistent-indexed-object-style)